kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/RealPostRepository.kt

```diff
@@ -21,29 +20,15 @@ internal class RealPostRepository(
     override suspend fun post(
         postId: String,
         postMetadataFields: String
-    ): DecoratedPost =
+    ): Post =
         withContext(dispatcher) {
```

Comment:
`apiService.post()` is already a `suspend` function, so there's no need to wrap this call within `withContext()`

Reply:
applied